dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogInterceptor.kt

```diff
@@ -41,11 +41,17 @@ import okhttp3.Response
  * request information (url, method, status code, optional error). It will also propagate the span
  * and trace information in the request header to link it with backend spans.
  *
+ * Note: If you want to get more insights on the network requests (e.g.: redirections), you can also add
```

Comment:
Not sure if you mean "for example" or "such as" or "that is" here, instead of `e.g.:` but you should use one of those phrases instead.